runelite-client/src/main/java/net/runelite/client/util/ImageUtil.java

```diff
@@ -326,6 +326,22 @@ public static BufferedImage outlineImage(final BufferedImage image, final Color
 		return outlineImage(image, color, ColorUtil::isNotFullyTransparent, outlineCorners);
 	}
 
+	/**
+	 * Outlines non-transparent pixels of a BufferedImage with the given color and given thickness. Optionally outlines
+	 * corners in addition to edges.
+	 *
+	 * @param image          The image to be outlined.
+	 * @param color          The color to use for the outline.
+	 * @param outlineCorners Whether to draw an outline around corners, or only around edges.
+	 * @param thickness      How thick the outline is.
+	 * @return               The BufferedImage with its edges--and optionally, corners--outlined
+	 *                       with the given color.
+	 */
+	public static BufferedImage outlineImage(final BufferedImage image, final Color color, final Boolean outlineCorners, final int thickness)
```

Comment:
Another overload of signature `BufferedImage image, Color color, int thickness` can be added to reduce verbisoty in usage.